docs/tutorial.md

```diff
@@ -0,0 +1,141 @@
+# Tutorial
+
+In this tutorial we will deploy the [sample express Node.js HTTP server](https://expressjs.com/en/starter/hello-world.html) in your own AWS account with AnyCloud. All the code can be found in this [template repository](https://github.com/alantech/hello-anycloud) which you can use to [create a new repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) for your AnyCloud project.
+
+## Enable programmatic AWS access for AnyCloud
+
+1) Create a new an IAM user in your AWS account using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).
+
+2) Create a new access key under that IAM user using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey).
+
+3) Enable programmatic access for that IAM user, and attach the built-in [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess)policy to it as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console).
+
+4) Take the `accessKeyId` and `secretAccessKey` from step 2 and add a local `~/.anycloud/credentials.json` file. The top-level key is the alias you provide for referring to these credentials later on in the `anycloud.json` file.
+
+```javascript
+{
+  "aws-personal": {
+    "cloudProvider": "AWS",
+    "credentials": {
+      "accessKeyId": "#####################",
+      "secretAccessKey": "###################"
+    },
```

Comment:
Still have a trailing comma that will fail JSON parsing. Also, can you change the ` ```javascript` to ` ```json` and see if that works? It does work on Github and it properly highlights invalid JSON for you:

```json
{
  "some: "JSON"
}
```

Reply:
yeah and `json` doesn't work on Gitbook

Reply:
```suggestion
    }
```